### PR TITLE
store mon and rgw secrets

### PR DESCRIPTION
**Pull request by travisn**
Rather than passing the secrets in plain text to the pods, the keyrings are stored in k8s secrets. #357

### CODE REVIEW

File: pkg/operator/mon/mon.go [hunk 1]
```diff
@@ -85,8 +89,47 @@ func (c *Cluster) Start(clientset *kubernetes.Clientset) (*mon.ClusterInfo, erro
...
+	}
+
+	// store the secrets
+	secrets := map[string]string{
+		clusterSecretName: info.Name,
+		fsidSecretName:    info.FSID,
+		monSecretName:     info.MonitorSecret,
+		adminSecretName:   info.AdminSecret,
+	}
+	_, err = clientset.Secrets(c.Namespace).Create(&v1.Secret{ObjectMeta: v1.ObjectMeta{Name: monSecretsName}, StringData: secrets})
```
Comment (jbw976) [inline pkg/operator/mon/mon.go]: we log the secrets when we find existing ones, should we be logging them when we create them too?

are there security concerns about writing the secrets to the log?

File: pkg/operator/rgw/rgw.go [hunk 2]
```diff
@@ -84,41 +86,59 @@ func (c *Cluster) Start(clientset *kubernetes.Clientset, cluster *mon.ClusterInf
...
 	}
 
-	return keyring, nil
+	// store the secrets
+	secrets := map[string]string{
+		rgwKeyringName: keyring,
+	}
+	_, err = clientset.Secrets(c.Namespace).Create(&v1.Secret{ObjectMeta: v1.ObjectMeta{Name: rgwSecretsName}, StringData: secrets})
+	if err != nil {
+		return fmt.Errorf("failed to save mon secrets. %+v", err)
```
Comment (jbw976) [inline pkg/operator/rgw/rgw.go]: copy/paste: should this be rgw secrets?

File: pkg/operator/mon/mon.go (hunk 1 above)
Comment (travisn) [inline pkg/operator/mon/mon.go]: bah, i intended to stop writing them to the log when i was done debugging. will remove

File: pkg/operator/rgw/rgw.go (hunk 2 above)
Comment (travisn) [inline pkg/operator/rgw/rgw.go]: good catch

**Pull request merged**